ui/src/shared/components/AnnotationTagEditorLi.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {PureComponent, ChangeEvent} from 'react'
+
+import {TagFilterType} from 'src/types/annotations'
+import {
+  Button,
+  ComponentColor,
+  ComponentSize,
+  ButtonShape,
+  IconFont,
+} from 'src/reusable_ui'
+
+interface Props {
+  tagKey: string
+  tagValue: string
+  onUpdate: (tagKey: string, tagValue: string) => void
+  onDelete: () => void
+}
+
+class AnnotationTagEditorLi extends PureComponent<Props> {
```

Comment:
this component name looks like a typo -- could you spell it out more specifically?